components/eamxx/src/dynamics/homme/eamxx_homme_process_interface.cpp

```diff
@@ -180,7 +180,7 @@ void HommeDynamics::set_grids (const std::shared_ptr<const GridsManager> grids_m
   add_field<Computed>("omega",              pg_scalar3d_mid, Pa/s,  pgn,N);
 
   add_tracer<Updated >("qv", m_phys_grid, kg/kg, N);
-  add_group<Updated>("tracers",pgn,N, true);
+  add_group<Updated>("tracers",pgn,N, Bundling::Required);
```

Comment:
Since we're bringing the enum back, I may as well ask: do you think the name `Bundling` is clear? Maybe we could change it into something else. Some ideas:
```c++
enum class GroupAllocation {
  Contiguous,
  NotContiguous
};
enum class MonolithicGroup {
  Required,
  NotRequired
};
enum class ContiguousGroup {
  Requred,
  NotRequired
};
```
I don't know. I wonder if the word "bundle" is a bit obscure, and we can find something more self-explanatory? If you have other name ideas, feel free to bounce them off us. This is the right time to discuss this, before we roll it out again.

Reply:
The user potentially needs to know what `group.m_bundle` is, maybe we would need to think about changing that name too? 

I like Monolithic the best. To me non-contiguous implies that "I can access the entire group, it just won't be contiguous" where the only way to access a view of the entire group is bundled case.

Reply:
Yes, if we decide on a name change, we should change it everywhere...

Reply:
What if we used
```c++
enum class MonolithicAlloc {
	Required,
	NotRequired
};
```
? I agree that the word "monolithic" carries a clear meaning. But I also like to have the word "allocation" (or something reminescent of it) somwhere.

What would you then call the bundle? `group.m_monolithic`? `group.m_monolithic_field`?  `group.m_whole`?  Something else? I like `m_monolithic_field`, and maybe rename `m_fields` into `m_individual_fields`?

Feel free to propose others...

Reply:
We can also merge as is, and follow up with a better name convention right after...

Reply:
I like `m_monolithic_field` and `m_individual_fields`. I think this is much clearer than "bundling". I've updated all the comments that refer to bundling as well.